tests/ui/transmute/unnecessary-transmute-path-remap-ice-140277.rs

```diff
@@ -0,0 +1,7 @@
+//@ aux-crate: zerocopy=unnecessary-transmute-path-remap-ice-140277-trans.rs
+#![deny(unnecessary_transmutes)]
```

Comment:
Problem: we never fire the lint against this combination in the first place, this should be `//@ check-pass`. Please also add a comment to describe the test intention, i.e. regression test for #140277 where there was a `span_to_snippet` invocation prior to checking if transmute is unnecessary.

Reply:
whoops i forgot to remove that